src/containers/apps/AppsTable.tsx

```diff
@@ -87,10 +86,11 @@ class AppsTable extends Component<
                 align: ALIGN,
                 sorter: (a, b) => {
                     return (
-                        new Date(b.lastDeployTime).getTime() -
-                        new Date(a.lastDeployTime).getTime()
+                        Date.parse(a.lastDeployTime) -
+                        Date.parse(b.lastDeployTime)
                     )
                 },
+                sortDirections: ['descend', 'ascend'],
```

Comment:
These changes keep the existing behaviour but fix the label when hovering the table header.